shared/src/commonMain/kotlin/io.projectnewm.shared/login/LoginService.kt

```diff
@@ -0,0 +1,38 @@
+package io.projectnewm.shared.login
+
+import io.ktor.client.*
+import io.ktor.client.request.*
+
+interface LoginService {
+    suspend fun requestSignUpAccess(email: String): String
+    suspend fun register(user: NewUser): String
+    suspend fun login(user: LoginUser): String
+
+    companion object {
+        const val BASE_URL = "https://staging-newm-server.herokuapp.com"
```

Comment:
Do we want this URL in a common location for reuse?

Reply:
We can find a better home for this once we implement other services and see what makes sense to refactor. 
Rule of three